LEAF_Request_Portal/Email.php

```diff
@@ -653,7 +653,7 @@ function attachApproversAndEmail($recordID, $emailTemplateID, $loggedInUser) {
                     // find the next step
                     $varsStep = array(':stepID' => $approvers[0]['stepID']);
                     $strSQL = "SELECT indicatorID_for_assigned_groupID FROM workflow_steps WHERE stepID=:stepID";
-                    $resStep = $this->db->prepared_query($strSQL, $varsStep);
+                    $resStep = $this->portal_db->prepared_query($strSQL, $varsStep);
```

Comment:
If portal-db is the correct name, what does db point to?

Reply:
It doesn't.  It's a bad copy and paste.  ;-)